src/main/java/pl/tomaszdziurko/jvm_bloggers/view/login/LoginPage.java

```diff
@@ -65,11 +75,11 @@ private void tryToLoginUser(String clientAddress) {
 
         CustomFeedbackPanel feedbackPanel = new CustomFeedbackPanel("feedbackPanel");
         loginForm.add(feedbackPanel);
-        RequiredTextField<String> loginField = new RequiredTextField<>(LOGIN_FIELD_ID);
+        RequiredTextField loginField = new RequiredTextField(LOGIN_FIELD_ID);
         loginForm.add(loginField);
         PasswordTextField passwordField = new PasswordTextField(PASSWORD_FIELD_ID);
         loginForm.add(passwordField);
-        Button loginButton =  new Button(FORM_SUBMIT_ID);
+        Button loginButton = new Button(FORM_SUBMIT_ID);
```

Comment:
Yes, you said it already, we heard ;) Even if these are not your changes you should leave the code in a better state than before your changes. Thats one of the rules of high quality software craftsmanship. Professionals should act in this way.